test/test_configure.rb

```diff
@@ -270,6 +270,44 @@ def test_time_partitioning
         assert_equal 'DAY', task['time_partitioning']['type']
       end
 
+      def test_range_partitioning
+        config = least_config.merge('range_partitioning' => {'field' => 'foo', 'range' => { 'start' => 1, 'end' => 2, 'interval' => 1 }})
```

Comment:
CI failed. Could you fix this?

```suggestion
        config = least_config.merge('range_partitioning' => {'field' => 'foo', 'range' => { 'start' => 1, 'end' => 3, 'interval' => 1 }})
```

```
===============================================================================
Failure: test_range_partitioning(Embulk::Output::Bigquery::TestConfigure):
  Exception raised:
  #<Embulk::ConfigError: `range_partitioning.range.start` + `range_partitioning.range.interval` must be less than `range_partitioning.range.end`>
/home/runner/work/embulk-output-bigquery/embulk-output-bigquery/test/test_configure.rb:275:in `test_range_partitioning'
     272: 
     273:       def test_range_partitioning
     274:         config = least_config.merge('range_partitioning' => {'field' => 'foo', 'range' => { 'start' => 1, 'end' => 2, 'interval' => 1 }})
  => 275:         assert_nothing_raised { Bigquery.configure(config, schema, processor_count) }
     276: 
     277:         # field is required
     278:         config = least_config.merge('range_partitioning' => {'range' => { 'start' => 1, 'end' => 2, 'interval' => 1 }})
```